cg/services/run_devices/pacbio/post_processing_service.py

```diff
@@ -49,11 +51,16 @@ def __init__(
         to_raise=PostProcessingError,
     )
     def post_process(self, run_name: str, dry_run: bool = False) -> None:
-        LOG.info(f"Starting PacBio post-processing for run: {run_name}")
+        LOG.info(f"Starting Pacbio post-processing for run: {run_name}")
         run_data: PacBioRunData = self.run_data_generator.get_run_data(
             run_name=run_name, sequencing_dir=self.sequencing_dir
         )
         self.run_validator.ensure_post_processing_can_start(run_data)
         self.store_service.store_post_processing_data(run_data=run_data, dry_run=dry_run)
         self.hk_service.store_files_in_housekeeper(run_data=run_data, dry_run=dry_run)
         self._touch_post_processing_complete(run_data)
+
+    def is_run_post_processed(self, run_name: str) -> bool:
```

Comment:
```suggestion
    def is_run_processed(self, run_name: str) -> bool:
```
Perhaps flavour but processed should be enough.

Reply:
fixed